controller/embedded/api-schema/definitions.all.json

```diff
@@ -1,257 +0,0 @@
-{
```

Comment:
All JSON schema is no longer needed. Replaced w/ go-swagger generated validation.